packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,931 @@
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/org/captcha"
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "enabledPages": [
+    "SSR",
+    "SSPR",
+    "SIGN_IN"
+  ],
+  "captchaId": "cap18c2Ey3iR9BLDb0g4",
+  "_links": {
+    "captcha": {
+      "href": "http://testorg.okta1.com:1802/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+      "hints": {
+        "allow": [
+          "GET",
+          "POST",
+          "PUT",
+          "DELETE"
+        ]
+      }
+    },
+    "self": {
+      "href": "http://testorg.okta1.com:1802/api/v1/org/captcha",
+      "hints": {
+        "allow": [
+          "GET",
+          "PUT"
+        ]
+      }
+    }
+  }
+}
+```
+
+This example returns an empty org-wide CAPTCHA settings:
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/org/captcha"
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "enabledPages": [],
+  "captchaId": null,
+  "_links": {
+    "self": {
+      "href": "http://testorg.okta1.com:1802/api/v1/org/captcha",
+      "hints": {
+        "allow": [
+          "GET",
+          "PUT"
+        ]
+      }
+    }
+  }
+}
+```
+
+### Update org-wide CAPTCHA settings
+
+<ApiOperation method="put" url="/api/v1/org/captcha" />
+
+Update org-wide CAPTCHA settings.
+
+> **Note:**  `captchaId` cannot be null if `enabledPages` is null or empty.
+
+#### Request path parameters
+
+N/A
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+The [Org-wide CAPTCHA Settings](#org-wide-captcha-settings-object) to update.
+
+#### Response body
+
+The updated [Org-wide CAPTCHA Settings response](#org-wide-captcha-settings-response-object)
+
+#### Use example
+
+This example updates org-wide CAPTCHA settings:
+
+##### Request
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+    "captchaId": "cap18c2Ey3iR9BLDb0g4",
+    "enabledPages": ["SSPR", "SIGN_IN"]
+}' "https://${yourOktaDomain}/api/v1/org/captcha"
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "enabledPages": [
+    "SSPR",
+    "SIGN_IN"
+  ],
+  "captchaId": "cap18c2Ey3iR9BLDb0g4",
+  "_links": {
+    "captcha": {
+      "href": "http://testorg.okta1.com:1802/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+      "hints": {
+        "allow": [
+          "GET",
+          "POST",
+          "PUT",
+          "DELETE"
+        ]
+      }
+    },
+    "self": {
+      "href": "http://testorg.okta1.com:1802/api/v1/org/captcha",
+      "hints": {
+        "allow": [
+          "GET",
+          "PUT"
+        ]
+      }
+    }
+  }
+}
+```
+
+This example disables CAPTCHA org-wide:
```

Comment:
Again, how? It disable it because it's setting the `captchaId` and `enabledpages` both to null?